app/lib/hooks/useKeyboardHeight.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+import { Keyboard } from 'react-native';
+
+// Taken from https://github.com/gie3d/react-native-use-keyboard-height/blob/master/src/index.tsx
+
+export const useKeyboardHeight = () => {
```

Comment:
We already have this hook on https://github.com/react-native-community/hooks#usekeyboard